vendor/github.com/99designs/gqlgen/codegen/generated!.gotpl

```diff
@@ -8,7 +8,7 @@
 {{ reserveImport "errors"  }}
 {{ reserveImport "bytes"  }}
 
-{{ reserveImport "github.com/vektah/gqlparser/v2" }}
+{{ reserveImport "github.com/vektah/gqlparser/v2" "gqlparser" }}
```

Comment:
are you/should we be worried about revendoring gqlgen and having this overwritten? I suppose it'll get better in december when we can start using modules